src/ui/save-html.js

```diff
@@ -63,38 +63,46 @@ const downloadLinks = [
   },
 ];
 
-function toDownloadLink(details) {
+/**
+ * @param {typeof downloadLinks[0]} details
+ */
+function toDownloadLink(details, conf) {
   const { id, href, fileName, title, type } = details;
+  const filename = conf.shortName
+    ? `${conf.shortName}-${ISODate.format(conf.publishDate || new Date())}`
```

Comment:
I think we want this:

```suggestion
    ? `${conf.specStatus}-${conf.shortName}-${concatDate(conf.publishDate || new Date())}`
```

So then it comes out like this:
![Screen Shot 2021-10-06 at 3 37 36 pm](https://user-images.githubusercontent.com/870154/136141488-b90d125d-2299-4b01-bb93-35375efbaa52.png)

Which was the original motivation.